syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -104,6 +104,10 @@
 }
 
 
+def is_running(x: str) -> bool:
```

Comment:
The signature and names are not super clear to me, could we use something closer to the intent such as `is_continue_decision(scheduler_decision: str) -> bool`?